doc/man7/EVP_RAND.pod

```diff
@@ -153,6 +153,36 @@ EVP_RAND_generate(<public>, ...) and
 EVP_RAND_generate(<private>, ...),
 respectively.
 
+When `[random] chain=0` is set in `openssl.cnf`, the public & private
+DRBG are not chained to primary, and instead use seed source
+directly. It is also possibly to configure OpenSSL to default to no
+chaining with -DOPENSSL_RAND_CHAIN=0.
+
+FIPS provider 3.5 and later are always unchained and use the following
```

Comment:
Since #25415 have been back-ported to 3.4 in #25526, should this be `FIPS provider 3.4 and later ...`?

Reply:
I totally missed that it was backported, my local branches were out of date. Let me re-ponder all this.